source/SpinalHDL/Libraries/stream.rst

```diff
@@ -425,10 +435,8 @@ This utile take its input stream and duplicate it outputCount times.
 .. code-block:: scala
 
    val inputStream = Stream(Bits(8 bits))
-   val dispatchedStreams = StreamDispatcherSequencial(
-     input = inputStream,
-     outputCount = 3
-   )
+   val (outputstream1, outputstream2) = StreamFork(inputStream,portCount=2)
```

Comment:
This seem broken to me, as StreamFork return a Vec[Stream[T]],
Shouldn't it be either : 

```scala
    val inputStream = Stream(Bool)
    val outputStreams = StreamFork(inputStream,portCount=2)
    // You get outputStreams(0) and outputStreams(1)
```

or 

```scala
    val inputStream = Stream(Bool)
    val (outputstream1, outputstream2)= StreamFork2(inputStream)
```

?

Reply:
Yes, wrongly changed from StreamFork2 to StreamFork without checking the return type would be different. Will change it.